docs/7___appendix.adoc

```diff
@@ -237,8 +233,9 @@ The integer part latexmath:[\mathbf{t}_I] of this tuple is a counter to enumerat
 |_super-dense time instant_
 |See <<time-instant>> and <<super-dense-time>>.
 
-|_task_
-|Special kind of _model partition_ that is used in control code.
+|[[task,task]]_task_
```

Comment:
If we can agree to erase "task" in docs/2_2_common_mechanisms.adoc then the term is only used in the SE example where it is stated as "A task can be thought of as a simple interface of the scheduler that allows to connect the FMU interface ...". Therefore I suggest to remove this definition which was my key intention when stating the related issue.

Reply:
removed in PR #1613